combat-trainer.lic

```diff
@@ -3069,7 +3081,7 @@ class GameState
   def determine_whirlwind_action
     if @use_weak_attacks && Flags['ct-accuracy-ready']
       update_analyze_array(true) if @analyze_combo_array.empty?
-      @analyze_combo_array.shift
+      @analyze_combo_array.empty? ? 'attack' : @analyze_combo_array.shift
```

Comment:
❓ Is it necessary to include code from the other PR? I'd prefer to make each change separately if possible.

Reply:
I'll remove the other PR stuff out